docs/design/alias-configuration.md

```diff
@@ -52,4 +69,56 @@ In this flow
+  - Recovered address must match with the alias (public address of the eth account)
+  - If the recovered address match with the address stored in the alias we have to update the key property of the account and store it in the ledger
+  - With the above step the process of lazy creation will be finished
+- Crypto Transfer Transaction to ECDSA key alias should create an account with its alias field set to the ECDSA key, populate the alias map with the EVM address calculated based on the ECDSA key, and the key field set with the value of the ECDSA key (currently implemented) and another Crypto Transfer Transaction to the EVM address corresponding to the ECDSA key should transfer to the account created in the previous transaction
+- Crypto Create Transaction with ED25519 key alias should create an account with its alias and key fields set to the ED25519 key. Attempting another Crypto Create Transaction with the same ED25519 key alias should fail
+- Crypto Create Transaction with ECDSA key alias:
+    
+**Always validate the 0.0.ECDSA alias format is missing from the alias map as well as the calculated evm_address alias format**
+    
+  1. Create an account with its alias and key fields set to the ECDSA key, and populate the alias map with the EVM address calculated based on the ECDSA key.
+      - Account with:
+          - hedera id: 0.0.20
+          - admin key: ECDSA key
+          - alias: 0.0.ECDSA key
+      - AliasManager map with entries:
+          - calculated EVM_address alias based on the ECDSA key → hedera id
+          - 0.0.ECDSA key → hedera id
+  
+  2. Attempting another Crypto Create Transaction with the same ECDSA key alias or EVM address corresponding to that key should fail
+    
+- Crypto Create Transaction with EVM address alias
+    1. Create an account with its alias field set to the EVM address and its key will be empty
+        - Account with:
+            - hedera id: 0.0.20
+            - admin key:
+            - alias: EVM_address alias
+        - AliasManager with:
+            - map with the EVM_address alias based on the ECDSA key → hedera id
+    2. a) When a signed transaction referencing the account by the EVM_address alias is received:
+        - Account with:
+            - hedera id: 0.0.20
+            - admin key: ECDSA key
+            - alias: 0.0.ECDSA key
+        - AliasManager with:
+            - map with the EVM_address alias based on the ECDSA key → hedera id
+            - map with the 0.0.ECDSA key alias → hedera id
+    
+    2. b) Don’t overwrite the alias field, instead add logic during the map rebuilding
+    - if the value of the alias field of the account has the evm_address format, add an entry to the aliases map with the value of 0.0.ECDSA key which is set on the account
+
+### Notes
+
+- For the EVM address cases we should store the alias value in the state not only in the memory structure
+- It is ok for different accounts to have the same key
```

Comment:
This is the case today. The AccountId is the real differentiator. Even if you have multiple accounts with the same public key only one can have it as the alias